frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterActionDropdown.tsx

```diff
@@ -205,6 +208,13 @@ export function ClusterActionDropdown(props: { cluster: Cluster; isKebab: boolea
 
     let actions = useMemo(
         () => [
+            {
+                id: 'update-automation-template',
+                text: t('Update automation template'),
+                click: (_cluster: Cluster) => setShowUpdateAutomationModal(true),
```

Comment:
Just remove the function parameter altogether. (Can do for the others below as well and remove the eslint-disable-next-line)